packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx

```diff
@@ -129,9 +133,9 @@ export function useActiveVisits() {
 
   return {
     activeVisits: formattedActiveVisits,
+    error,
     isLoading,
     isValidating,
```

Comment:
Renames `isError` to `error`, more accurately representing what the variable does.